CHANGELOG.md

```diff
@@ -28,7 +26,7 @@
 ## Deprecation Notices
 
 - Deprecated ~~`$compileProvider.aHrefSanitizationWhitelist`~~.
-  It is now [aHrefSanitizationTrustedUrlList](https://docs.angularjs.org/api/ng/provider/$compileProvider#aHrefSanitizationTrustedUrlList)`.
+  It is now [aHrefSanitizationTrustedUrlList](https://docs.angularjs.org/api/ng/provider/$compileProvider#aHrefSanitizationTrustedUrlList).
```

Comment:
I'd suggest using backticks for the name, this would be more compatible with how the replaced APIs are replaced. E.g. here:
```suggestion
  It is now [`aHrefSanitizationTrustedUrlList`](https://docs.angularjs.org/api/ng/provider/$compileProvider#aHrefSanitizationTrustedUrlList).
```
Similar changes can be done for all other lines here.